internal/cli/default_setter_opts.go

```diff
@@ -42,15 +42,17 @@ type ProjectOrgsLister interface {
 }
 
 type DefaultSetterOpts struct {
-	Service          string
-	OpsManagerURL    string
-	ProjectID        string
-	OrgID            string
-	MongoShellPath   string
-	TelemetryEnabled bool
-	Output           string
-	Store            ProjectOrgsLister
-	OutWriter        io.Writer
+	Service                string
+	OpsManagerURL          string
+	ProjectID              string
+	OrgID                  string
+	MongoShellPath         string
+	TelemetryEnabled       bool
+	Output                 string
+	Store                  ProjectOrgsLister
+	OutWriter              io.Writer
+	AskedOrgsOrProjects    bool
+	AskOrgsAndProjectsText string
```

Comment:
[optional]
AskOrgsAndProjectsText to control the flow (not only as a label) which is a bit confusing.
What do you think about this alternative:
- instead of AskOrgsAndProjectsText have a field `OnMultipleOrgsOrProjects func () void`
- set this field in in login.go (instead of setting opts.AskOrgsAndProjectsText) with the content from displayInitialSelectText()
- in AskOrg / AskProject instead of calling opts.displayInitialSelectText(), call opts.OnMultipleOrgsOrProjects() (if it's not nil)

Reply:
Yes, that seems like a better extensible approach! Thank you